src/en/foundations/colour/usage.md

```diff
@@ -12,31 +12,66 @@ eleventyNavigation:
   alt: This is an image of the foundation
 permalink: /en/foundations/colour/
 translationKey: "colour"
-tags: ['colourEN', 'usage']
+tags: ["colourEN", "usage"]
+type: color
```

Comment:
type is the token type. It can be overwritten when calling the partial if needed